articles/quickstart.md

```diff
@@ -178,149 +144,149 @@ Add the following operation to the namespace, after the end of the
+
+## Create the host application code
+
+#### [Python](#tab/tabid-python)
+
+1. Open the `host.py` file and add the following code:
+
+    ```python
+    import qsharp
+
+    from qsharp import Result
+    from Quantum.Bell import BellTest
+
+    initials = (Result.Zero, Result.One)
+
+    for i in initials:
+      res = BellTest.simulate(count=1000, initial=i)  
+      (num_zeros, num_ones) = res
+      print(f'Init:{i: <4} 0s={num_zeros: <4} 1s={num_ones: <4}')
+    ```
+
+#### [C#](#tab/tabid-csharp)
+
+1. Replace the contents of the `Driver.cs` file with the following code:
+
+    ```csharp
+    using System;
+
+    using Microsoft.Quantum.Simulation.Core;
+    using Microsoft.Quantum.Simulation.Simulators;
+
+    namespace Quantum.Bell
     {
-        static void Main(string[] args)
+        class Driver
         {
-            using (var qsim = new QuantumSimulator())
+            static void Main(string[] args)
             {
-                HelloQ.Run(qsim).Wait();
+                using (var qsim = new QuantumSimulator())
+                {
+                    // Try initial values
+                    Result[] initials = new Result[] { Result.Zero, Result.One };
+                    foreach (Result initial in initials)
+                    {
+                        var res = BellTest.Run(qsim, 1000, initial).Result;
+                        var (numZeros, numOnes) = res;
+                        System.Console.WriteLine(
+                            $"Init:{initial,-4} 0s={numZeros,-4} 1s={numOnes,-4}");
+                    }
+                }
+
+                System.Console.WriteLine("Press any key to continue...");
+                Console.ReadKey();
             }
         }
     }
-}
-```
+    ```
 
-#### Run quantum operations from a C# host program
-Replace the body of the `Main` method with the following code:
+#### [](#tab/tabid-vs2019)
 
-```csharp
-            using (var qsim = new QuantumSimulator())
-            {
-                // Try initial values
-                Result[] initials = new Result[] { Result.Zero, Result.One };
-                foreach (Result initial in initials)
-                {
-                    var res = BellTest.Run(qsim, 1000, initial).Result;
-                    var (numZeros, numOnes) = res;
-                    System.Console.WriteLine(
-                        $"Init:{initial,-4} 0s={numZeros,-4} 1s={numOnes,-4}");
-                }
-            }
+* * *
 
-            System.Console.WriteLine("Press any key to continue...");
-            Console.ReadKey();
-```
+### About the host application code
+
+#### [Python](#tab/tabid-python)
+
+The Python host application has three parts:
+
+* Compute any arguments required for the quantum algorithm. In the example, `count` is fixed at a 1000 and `initial` is the initial value of the qubit.
```

Comment:
This was actually another nuance I forgot to mention in the call yesterday, that might interest @cgranade too. By way of fumbling and bumbling I found that putting "0" and "1" in as the initial values in the Python call works. I learned the hard way after trying the "Result.Zero"/"Result.One" without knowing to import "Result" from qsharp as you did here--so thank you, I'll change to that in my tutorial.

Chris, in particular: during that fumbling, I learned that any input like "0.5" or "2" also worked without throwing any errors (though the output wasn't what I expected). I didn't look for a pattern or anything, so I'm not sure if it's a bug or something weird but expected, but figured I'd give you a heads up anyhow.

Reply:
@gillenhaalb: Would you be willing to file that as a bug on microsoft/iqsharp? That's good to know, thank you!

Reply:
Just put it up, no problem!

Reply:
Thanks! Resolving the conversation here, then, as I don't think there's anything further that can be actioned on that from within this PR.